.travis.yml

```diff
@@ -39,7 +39,7 @@ install:
   - pip install --upgrade appdirs # https://github.com/ActiveState/appdirs/issues/89#issuecomment-282326570
   - python setup.py egg_info
   - pip install --upgrade --editable .
-  - pip install --upgrade --requirement docs/requirements.txt
+  - pip install --upgrade --requirement travis/docker/requirements.txt
```

Comment:
What is this rename for?

Reply:
See comments in travis/docker/requirements.txt above. I am reverting it back to docs/requirements.txt.